firefox-ios/WidgetKit/DownloadManager/DownloadLiveActivity.swift

```diff
@@ -83,3 +96,87 @@ struct DownloadLiveActivity: Widget {
         }
     }
 }
+
+@available(iOS 16.2, *)
+@DynamicIslandExpandedContentBuilder
+private func expandedContent
+(liveDownload: ActivityViewContext<DownloadLiveActivityAttributes>, settings: expandedContentSettings) ->
+DynamicIslandExpandedContent<some View> {
+        DynamicIslandExpandedRegion(.leading) {
+            ZStack {
+                RoundedRectangle(cornerRadius: settings.iconEdgeRounding)
+                    .fill(settings.widgetColours)
+                    .frame(width: settings.iconFrameSize,
+                           height: settings.iconFrameSize)
+                Image(settings.firefoxIcon)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: settings.firefoxIconSize,
+                           height: settings.firefoxIconSize)
+            }.padding(EdgeInsets(top: settings.iconTopPadding,
+                                 leading: settings.iconLeftPadding,
+                                 bottom: settings.iconBottomPadding,
+                                 trailing: settings.iconRightPadding))
+        }
+        DynamicIslandExpandedRegion(.trailing) {
+            ZStack {
+                Circle()
+                    .stroke(settings.widgetColours.opacity(settings.inProgessOpacity),
+                            lineWidth: settings.progressWidth)
+                    .frame(width: settings.iconFrameSize,
+                           height: settings.iconFrameSize)
+                Circle()
+                    .trim(from: 0.0,
+                          to: min(liveDownload.state.totalProgress, 1.0))
+                    .stroke(style: StrokeStyle(lineWidth: settings.progressWidth))
+                    .rotationEffect(.degrees(270.0))
+                    .animation(.linear, value: 0.5)
+                if liveDownload.state.totalProgress == 1.0 {
+                    Image(settings.checkmarkIcon)
+                    .renderingMode(.template)
+                    .frame(width: settings.stateIconSize,
+                           height: settings.stateIconSize)
+                    .foregroundStyle(settings.widgetColours)
+                } else {
+                    Image(settings.mediaStopIcon)
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: settings.stateIconSize,
+                               height: settings.stateIconSize)
+                }
+            }.frame(width: settings.iconFrameSize,
+                    height: settings.iconFrameSize)
+                .padding(EdgeInsets(top: settings.iconTopPadding,
+                                    leading: settings.iconLeftPadding,
+                                    bottom: settings.iconBottomPadding,
+                                    trailing: settings.iconRightPadding))
+        }
+        DynamicIslandExpandedRegion(.center) {
+            Text("Downloading \(liveDownload.state.downloads[0].fileName)")
```

Comment:
We should use [localized](https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Shared/Strings.swift#L7898-L7913) strings here  ( this should take care of the linter failure in the checks 😄  )
```suggestion
            Text(String(format: .LiveActivity.Downloads.FileNameText, liveDownload.state.downloads[0].fileName))
```